lib/helpers.js

```diff
@@ -18,13 +18,19 @@ var logger = require('./server/logger.js').get('appium')
 exports.downloadFile = function (fileUrl, suffix, cb) {
   // We will be downloading the files to a directory, so make sure it's there
   // This step is not required if you have manually created the directory
+  cb = _.once(cb);
```

Comment:
is this just to be safe? Or is there a specific code path we know would end in this being called multiple times.